code/modules/jobs/job_types/roguetown/refugees/types/ecclesiast.dm

```diff
@@ -96,34 +97,215 @@
+					H.mind.adjust_skillrank(/datum/skill/misc/climbing, 1, TRUE)
+					pants = /obj/item/clothing/under/roguetown/tights/black
+					shoes = /obj/item/clothing/shoes/roguetown/armor
+					gloves = /obj/item/clothing/gloves/roguetown/leather
+					belt = /obj/item/storage/belt/rogue/leather
+					shirt = /obj/item/clothing/suit/roguetown/shirt/undershirt/random
+					armor = /obj/item/clothing/suit/roguetown/armor/plate/half/iron
+					neck = /obj/item/clothing/neck/roguetown/gorget
+					backl = /obj/item/storage/backpack/rogue/satchel
+					beltr = /obj/item/rogueweapon/sword/iron
+					beltl = /obj/item/storage/belt/rogue/pouch/coins/poor
+					H.change_stat("strength", 4)
+					H.change_stat("endurance", 1)
+					H.change_stat("constitution", 1)
+					ADD_TRAIT(H, TRAIT_MEDIUMARMOR, TRAIT_GENERIC)
+					var/datum/devotion/C = new /datum/devotion(H, H.patron)
+					C.grant_spells(H)
+					H.verbs += list(/mob/living/carbon/human/proc/devotionreport, /mob/living/carbon/human/proc/clericpray)
+				if(/datum/patron/divine/necra)
+					H.mind.adjust_skillrank(/datum/skill/misc/athletics, 3, TRUE)
+					H.mind.adjust_skillrank(/datum/skill/magic/holy, 3, TRUE)
+					H.mind.adjust_skillrank(/datum/skill/combat/unarmed, 4, TRUE)
+					H.mind.adjust_skillrank(/datum/skill/combat/wrestling, 4, TRUE)
+					H.mind.adjust_skillrank(/datum/skill/misc/reading, 2, TRUE)
+					H.mind.adjust_skillrank(/datum/skill/misc/climbing, 1, TRUE)
+					H.mind.adjust_skillrank(/datum/skill/misc/swimming, 1, TRUE)
+					H.mind.adjust_skillrank(/datum/skill/misc/treatment, 3, TRUE)
+					pants = /obj/item/clothing/under/roguetown/tights/black
+					shoes = /obj/item/clothing/shoes/roguetown/armor
+					gloves = /obj/item/clothing/gloves/roguetown/plate
+					belt = /obj/item/storage/belt/rogue/leather
+					shirt = /obj/item/clothing/suit/roguetown/shirt/undershirt/random
+					armor = /obj/item/clothing/suit/roguetown/armor/plate/half/iron
+					neck = /obj/item/clothing/neck/roguetown/gorget
+					backl = /obj/item/storage/backpack/rogue/satchel
+					beltl = /obj/item/storage/belt/rogue/pouch/coins/poor
+					H.change_stat("strength", 2)
+					H.change_stat("endurance", 2)
+					H.change_stat("constitution", 2)
+					ADD_TRAIT(H, TRAIT_MEDIUMARMOR, TRAIT_GENERIC)
+					var/datum/devotion/C = new /datum/devotion(H, H.patron)
+					C.grant_spells(H)
+					H.verbs += list(/mob/living/carbon/human/proc/devotionreport, /mob/living/carbon/human/proc/clericpray)
+				if(/datum/patron/divine/malum)
+					H.mind.adjust_skillrank(/datum/skill/combat/maces, 3, TRUE)
+					H.mind.adjust_skillrank(/datum/skill/misc/athletics, 2, TRUE)
+					H.mind.adjust_skillrank(/datum/skill/magic/holy, 3, TRUE)
+					H.mind.adjust_skillrank(/datum/skill/misc/reading, 2, TRUE)
+					H.mind.adjust_skillrank(/datum/skill/misc/climbing, 1, TRUE)
+					H.mind.adjust_skillrank(/datum/skill/misc/swimming, 1, TRUE)
+					H.mind.adjust_skillrank(/datum/skill/craft/blacksmithing, 3, TRUE)
```

Comment:
We're not having smithing clerics. Malum oriented clerics are NOT getting smithing skill round start